app/scripts/pages/config.js

```diff
@@ -9,6 +9,8 @@ var Config = React.createClass({
   getInitialState: function () {
     var t = this
     t.props.ipfs.config.show(function (err, configStream) {
+      if (err) console.log(err)
```

Comment:
no return statement here? 

btw, we can definitely have more logs for debug mode in browser things we do. I really like [bows](https://www.npmjs.com/package/bows) for logging in browser apps, do you have any that you like more?